controllers/apply/expiries/send-expiry-email.js

```diff
@@ -0,0 +1,177 @@
+                en: 'You have one day to finish your funding proposal',
+                cy: 'Mae gennych ddiwrnod ar ôl i orffen eich cynnig'
+            };
+            break;
+    }
+
+    if (isBilingual) {
+        return [subjectLine.en, subjectLine.cy].join(' / ');
+    } else {
+        return subjectLine.en;
+    }
+}
+
+function getEmailFor(country) {
+    const countryEmail = {
+        'scotland': 'advicescotland@tnlcommunityfund.org.uk',
+        'northern-ireland': 'enquiries.ni@tnlcommunityfund.org.uk',
+        'wales': 'wales@tnlcommunityfund.org.uk'
+    }[country];
+
+    return countryEmail || 'general.enquiries@tnlcommunityfund.org.uk';
+}
+
+function getPhoneFor(country) {
+    const countryPhone = {
+        'scotland': '0300 123 7110',
+        'northern-ireland': '028 9055 1455',
+        'wales': '0300 123 0735'
+    }[country];
+
+    return countryPhone || '0345 4 10 20 30';
+}
+
+module.exports = function sendExpiryEmail(
+    {
+        formId,
+        emailType,
+        unsubscribeToken,
+        applicationId,
+        applicationData = {},
+        expiresAt,
+        sendTo
+    },
+    mockMailTransport = null
+) {
+    const emailConfig = {
+        'awards-for-all': {
+            name: 'application_expiry_afa',
+            template: path.resolve(
+                __dirname,
+                './views/awards-for-all-expiry-email.njk'
+            )
+        },
+        'standard-enquiry': {
+            name: 'application_expiry_standard',
+            template: path.resolve(
+                __dirname,
+                './views/standard-expiry-email.njk'
+            )
+        }
+    }[formId];
+
+    function getProjectCountry() {
+        if (formId === 'awards-for-all') {
+            return get(applicationData, 'projectCountry');
+        } else if (formId === 'standard-enquiry') {
+            const countries = get(applicationData, 'projectCountries');
+            return countries && countries.length === 1
+                ? countries[0]
+                : 'Multiple';
+        }
+    }
+
+    function getBilingualStatus() {
+        if (formId === 'awards-for-all') {
+            return get(applicationData, 'projectCountry') === 'wales';
+        } else if (formId === 'standard-enquiry') {
+            const countries = get(applicationData, 'projectCountries');
+            return countries && countries.includes('wales');
+        }
+    }
+
+    function getExpiryDates() {
+        const expiresOn = moment(expiresAt);
+        const dateFormat = 'D MMMM, YYYY HH:mm';
+        return {
+            en: expiresOn.format(dateFormat),
+            cy: expiresOn.locale('cy').format(dateFormat)
+        };
+    }
+
+    const baseUrl = 'https://www.tnlcommunityfund.org.uk';
+
+    const unsubscribeLinkUrlPath = `/apply/emails/unsubscribe?token=${unsubscribeToken}`;
+    const unsubscribeLink = {
+        en: `${baseUrl}${unsubscribeLinkUrlPath}`,
+        cy: `${baseUrl}welsh${unsubscribeLinkUrlPath}`
```

Comment:
Is this missing a slash still? eg. won't it output `https://www.tnlcommunityfund.org.ukwelsh/apply/emails/unsubscribe?`?